tools/sigclient/pkg/utils/reader.go

```diff
@@ -499,9 +594,41 @@ func randomizeBody_functionalTest(f *gofakeit.Faker, m map[string]interface{}, a
 	}
 }
 
+func randomizeBody_perfTest(f *gofakeit.Faker, m map[string]interface{}, addts bool, config *GeneratorDataConfig, accountFaker *gofakeit.Faker) {
+	randomizeBody_functionalTest(f, m, addts, config.functionalTest, accountFaker)
+
+	if !config.perfTestConfig.SendData {
```

Comment:
nit: Consider renaming this as `ShouldSendData`. But if `SendData` makes more sense to you, then you can keep that as well.

Reply:
addressed in the latest commit.